src/User/resources/i18n/de/usuario.php

```diff
@@ -17,75 +17,103 @@
  * NOTE: this file must be saved in UTF-8 encoding.
  */
 return [
-    'An email with instructions to create a new password has been sent to {email} if it is associated with an {appName} account. Your existing password has not been changed.' => '',
-    'Are you sure you wish the user to change their password at next login?' => '',
-    'Back to privacy settings' => '',
-    'Data processing consent' => '',
-    'Delete my account' => '',
-    'Delete personal data' => '',
-    'Deleted by GDPR request' => '',
-    'Disable two factor authentication' => '',
-    'Download my data' => '',
-    'Enable two factor authentication' => '',
-    'Export my data' => '',
-    'Force password change at next login' => '',
-    'Here you can download your personal data in a comma separated values format.' => '',
-    'I agree processing of my personal data and the use of cookies to facilitate the operation of this site. For more information read our {privacyPolicy}' => '',
-    'Invalid password' => '',
-    'Invalid two factor authentication code' => '',
-    'Last login IP' => '',
-    'Last login time' => '',
-    'Last password change' => '',
-    'Once you have deleted your data, you will not longer be able to sign in with this account.' => '',
-    'Password age' => '',
-    'Privacy' => '',
-    'Privacy settings' => '',
-    'Rule class name' => '',
-    'Select rule...' => '',
-    'There was an error in saving user' => '',
-    'This will disable two factor authentication. Are you sure?' => '',
-    'This will remove your personal data from this site. You will no longer be able to sign in.' => '',
-    'Two Factor Authentication (2FA)' => '',
-    'Two factor authentication code' => '',
-    'Two factor authentication has been disabled.' => '',
-    'Two factor authentication protects you in case of stolen credentials' => '',
-    'Two factor authentication successfully enabled.' => '',
-    'Unable to disable Two factor authentication.' => '',
-    'User will be required to change password at next login' => '',
-    'You are about to delete all your personal data from this site.' => '',
-    'Your consent is required to register' => '',
-    'Your password has expired, you must change it now' => '',
-    'Your personal information has been removed' => '',
-    'privacy policy' => '',
+    'An email with instructions to create a new password has been sent to {email} if '.
+        'it is associated with an {appName} account. Your existing password has not been changed.'
+        => 'Eine Email mit Hinweisen, wie ein Kennwort erstellt wirdm wurde an {email} gesendet, '.
```

Comment:
wirdm? You mean wird :)

Reply:
oh, indeed. i reviewd all translations and corrected the typos.